spk/demoservice/src/conf/resource

```diff
@@ -0,0 +1,17 @@
+{
+  "mysql-db": {
```

Comment:
```suggestion
  "mysql-db": {
  "mariadb10-db" {
```

I contacted Synology a while back, and they gave me updated documentation. I assumed it was already updated online as well. [developer-guide-ex.pdf](https://github.com/SynoCommunity/spksrc/files/6164396/developer-guide-ex.pdf)

Also see 

https://github.com/SynoCommunity/spksrc/blob/dsm7-packages/spk/wallabag/src/conf/resource

https://github.com/SynoCommunity/spksrc/wiki/DSM-7.0-Beta#package-migration